.github/workflows/docker-image.yml

```diff
@@ -16,6 +16,7 @@ jobs:
           - image: fedora34
           - image: fedora37
           - image: ubuntu1804
```

Comment:
Everything in this list is being built... since we only want to build the new image we can remove everything else from the list